tidb-cloud/terraform/tidbcloud-provider.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Get TiDB Cloud Terraform Provider
+summary: Learn how to get TiDB Cloud Terraform provider
+---
+
+# Get TiDB Cloud Terraform Provider
+
+You will learn how to get TiDB Cloud Terraform provider in this doc.
+
+> **Note:**
+>
+> Before you begin, make sure that the requirements in [overview](/tidb-cloud/terraform/terraform-overview.md) are met.
+
+## Set up terraform
```

Comment:
```suggestion
## Set up Terraform
```

Reply:
reason: If users need to complete all the steps in this section by sequence, we can use Step 1, Step 2, Step 3 to present the section titles to make them more clear.